kuma/wiki/templates/wiki/document.html

```diff
@@ -58,8 +59,8 @@
   <link rel="alternate" type="application/json" href="{{ url('wiki.json_slug', document.slug) }}">
   <link rel="canonical" href="{{ canonical }}" >
 
-  {% if is_translated %}
-    {% for translation in document.other_translations %}
+  {% if other_translations %}
```

Comment:
It seems return some kind of truthy object, because the spot check works fine here and for [the other boolean check for the lang switcher down on line 81](https://github.com/groovecoder/kuma/commit/8795d34d3aff5eb2a51c5f442c1bf1b764b88f06#diff-09328eed22f2f3060a415388b98060ccR81)

I removed the call to `exists` because [the docs](https://docs.djangoproject.com/en/1.7/ref/models/querysets/#django.db.models.query.QuerySet.exists) say...
> if some_queryset has not yet been evaluated, but you know that it will be at some point, then using some_queryset.exists() will do more overall work (one query for the existence check plus an extra one to later retrieve the results) than simply using bool(some_queryset), which retrieves the results and then checks if any were returned.
We can't call `bool` directly in jinja, but if we want to be explicit, I could add a `has_translations = bool(doc.other_translations)` type casting call to the view and pass it along to the context?